### PR TITLE
Doc Review Updates

### PR DESCRIPTION
**Pull request by wilwade**
Fixes to the docs, both in the markdown and in the swagger and comments

- Grammar fixes
- Wording fixes
- OpenAPI consistency
- "Sign In With Frequency" (no dash, etc..)
- "MSA Id" in text `msaId` or `xMsaId in code
- etc...

NOT in this PR: Removing `DTO/Dto` from names

## Steps to Verify:

- `make format`, `make generate-openapi` should have no changes remaining
- `cd docs && mdbook serve`

### CODE REVIEW

File: docs/src/Build/README.md [hunk 1]
```diff
@@ -11,39 +11,46 @@
 
 ## Account Service
 
-Account Service is a service enabling easy interaction with accounts on Frequency.
-Accounts are be defined as an `msaId` (a 64 bit identifier) and can contain additional information such as a handle, keys, and more.
+The Account Service enables easy interaction with accounts on Frequency.
+Accounts are defined as an `msaId` (64-bit identifier) and can contain additional information such as a handle, keys, and more.
```
Comment (JoeCap08055) [inline docs/src/Build/README.md:15]: Should we say anything about MSA identifiers being represented as string values in the API, not actual numeric fields?

File: docs/src/Build/README.md [hunk 2]
```diff
@@ -11,39 +11,46 @@
...
-- Asset creation with [IPFS](https://ipfs.tech)
+- Store and attach media with [IPFS](https://ipfs.tech)
+
+See [Content Publishing Service Details & API Reference](./ContentPublishing)
 
 ## Content Watcher Service
 
-The Content Watcher Service is a service enabling processing external content from Frequency.
-Feed content through your custom webhooks to connect to the larger ecosystem of content.
+The Content Watcher Service enables the processing of content external from Frequency and feeding content through custom webhooks to connect to the larger ecosystem of content.
```
Comment (JoeCap08055) [inline docs/src/Build/README.md:50]:
```suggestion
The Content Watcher Service enables client applications to register for traditional webhook notifications when relevant content is found on Frequency, without needing to interact with the chain themselves.
```

File: docs/src/Fundamentals/README.md [hunk 3]
```diff
@@ -6,12 +6,12 @@
 
 Frequency provides a shared global state to make interoperability and user control fundamental to the internet.
 Applications provide unique experiences to their users while accessing the content and graph connections from other applications.
-Just like how the internet is a shared global network, your applications can break out of their artificial boundaries.
+In the same way the internet is a shared global network, Frequency allows your applications to move past artificial boundaries.
```
Comment (JoeCap08055) [inline docs/src/Fundamentals/README.md:9]: nit: prioritize Frequency in the messaging. Also, if we're making comparisons, perhaps the network stack itself is more apt than just "the internet"?
```suggestion
Frequency enables your applications to interact with this shared global state seamlessly, in the same way that modern networking software allowed isolated computers to interact seamlessly over a global network.
```

File: docs/src/GettingStarted/README.md [hunk 4]
```diff
@@ -20,11 +20,11 @@ Before you begin, ensure you have the following installed on your machine:
...
 This will enable users to delegate to you, and your chain actions to be free via [Capacity](https://docs.frequency.xyz/Tokenomics/ProviderIncentives.html).
 
-### Create an Application Wallet
+### Create an Application Account in a Wallet
 
-- Open Polkadot Extension
+- Open a wallet extension such as the [Polkadot extension](https://polkadot.js.org/extension/)
 - Follow account creation steps
-- Make sure to keep the seed phrase for service configuration step
+- _Make sure to keep the seed phrase for service configuration step_
```
Comment (JoeCap08055) [inline docs/src/GettingStarted/README.md:27]:
```suggestion
- _Make sure to keep the seed phrase for the service configuration step_
```

File: docs/src/README.md [hunk 5]
```diff
@@ -26,17 +26,16 @@ Gateway provides a simple API to interact with the Frequency social layers of id
...
 ![Gateway Application Microservice Diagram](./gateway_arch-TopLevelServices.drawio.png)
 
 ## Key Microservices
 
 ### Account Service
 
-Account Service is a service enabling easy interaction with accounts on Frequency.
-Accounts are be defined as an `msaId` (a 64 bit identifier) and can contain additional information such as a handle, keys, and more.
+The Account Service enables easy interaction with accounts on Frequency.
+Accounts are defined as an `msaId` (64-bit identifier) and can contain additional information such as a handle, keys, and more.
```
Comment (JoeCap08055) [inline docs/src/README.md:38]: see previous note about MSA IDs as strings

File: docs/src/README.md [hunk 6]
```diff
@@ -54,19 +53,18 @@ Each Graph connection on Frequency can be private or public and can be unidirect
...
 - Create updates to existing content
 - Request deletion of content
-- Media attachement storage with [IPFS](https://ipfs.tech)
+- Store and attach media with [IPFS](https://ipfs.tech)
 
 ### Content Watcher Service
 
-The Content Watcher Service is a service enabling processing external content from Frequency.
-Feed content through your custom webhooks to connect to the larger ecosystem of content.
+The Content Watcher Service enables the processing of content external from Frequency and feeding content through custom webhooks to connect to the larger ecosystem of content.
```
Comment (JoeCap08055) [inline docs/src/README.md:67]: see previous note; keep in sync here

File: docs/src/Support/Channels.md [hunk 7]
```diff
@@ -1,7 +1,7 @@
 # Support Channels
 
 Need help with Gateway or working through the Frequency ecosystem?
-Lots of different options depending on your need.
+There are lots of different options to meet your needs.
```
Comment (JoeCap08055) [inline docs/src/Support/Channels.md:4]:
```suggestion
There are many different options to meet your needs.
```

File: services/content-publishing/env.template [hunk 8]
```diff
@@ -36,14 +36,14 @@ API_PORT=3000
...
 
-# Number of seconds between content publishing batches. This is so that
-# the service waits a reasonable amount of time for additional content to publishing
-# before submitting a batch--it represents a trade-off between maximum batch fullness
-# and minimal wait time for published content.
+# Number of seconds between publishing batches.
+# This is so that the service waits a reasonable amount of time for
+# additional content to publish before submitting a batch--it represents
+# a trade-off between maximum batch fullness and minimal wait time for content
 BATCH_INTERVAL_SECONDS=12
```
Comment (JoeCap08055) [inline services/content-publishing/env.template:47]: not related to documentation, but I wonder if we can eliminate this config--might make sense to just auto-calculate it based on `SECONDS_PER_BLOCK`

**Review by JoeCap08055 — COMMENTED** [review]
Some non-blocking comments, plus some reminders:
1. Be sure to regenerate all services' `swagger.json` *AND webhook OpenAPI types* before merging
2. After merging, need a PR in `social-app-template` repo to update all of the generated openapi interfaces

File: docs/src/Build/README.md (hunk 1 above)
Comment (wilwade) [reply docs/src/Build/README.md:15]: in JSON, strings are the only safe transport for 64bit ints, so I think it is fine as is.

File: services/content-publishing/env.template (hunk 8 above)
Comment (wilwade) [reply services/content-publishing/env.template:47]: Might be better for the programming, but I think keeping it in seconds makes it easier to think about for the user (dev in this case)

Comment (JoeCap08055) [reply services/content-publishing/env.template:47]: What I mean is, do we even need this as an independently-configurable parameter? Or should it just be calculated to be some relationship to the block time?

Again, not relevant to this PR, but perhaps for later discussion.

**Pull request merged**